clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelPropertiesDescriptionListGroup.tsx

```diff
@@ -49,88 +49,110 @@ const ModelPropertiesDescriptionListGroup: React.FC<ModelPropertiesDescriptionLi
   );
 
+  // State to control the ExpandableSection itself, default to expanded
+  const [isPropertiesSectionExpanded, setIsPropertiesSectionExpanded] = React.useState(true);
+
   return (
     <DashboardDescriptionListGroup
-      title="Properties"
-      action={
-        !isArchive && (
-          <Button
-            variant="link"
-            data-testid="add-property-button"
-            icon={<PlusCircleIcon />}
-            iconPosition="start"
-            isDisabled={isAdding || isSavingEdits}
-            onClick={() => {
-              setIsShowingMoreProperties(true);
-              setIsAdding(true);
-            }}
-          >
-            Add property
-          </Button>
-        )
-      }
+      title=""
       isEmpty={!isAdding && keys.length === 0}
       contentWhenEmpty="No properties"
     >
-      <Table aria-label="Properties table" data-testid="properties-table" variant="compact">
-        <Thead>
-          <Tr>
-            <Th>Key {isEditingSomeRow && requiredAsterisk}</Th>
-            <Th>Value {isEditingSomeRow && requiredAsterisk}</Th>
-            <Th screenReaderText="Actions" />
-          </Tr>
-        </Thead>
-        <Tbody>
-          {shownKeys.map((key) => (
-            <ModelPropertiesTableRow
-              key={key}
-              isArchive={isArchive}
-              keyValuePair={{ key, value: filteredProperties[key].string_value }}
-              allExistingKeys={allExistingKeys}
-              isEditing={editingPropertyKeys.includes(key)}
-              setIsEditing={(isEditing) => setIsEditingKey(key, isEditing)}
-              isSavingEdits={isSavingEdits}
-              setIsSavingEdits={setIsSavingEdits}
-              saveEditedProperty={(oldKey, newPair) =>
-                saveEditedCustomProperties(
-                  mergeUpdatedProperty({ customProperties, op: 'update', oldKey, newPair }),
-                )
-              }
-              deleteProperty={(oldKey) =>
-                saveEditedCustomProperties(
-                  mergeUpdatedProperty({ customProperties, op: 'delete', oldKey }),
-                )
-              }
-            />
-          ))}
-          {isAdding && (
-            <ModelPropertiesTableRow
-              isAddRow
-              allExistingKeys={allExistingKeys}
-              setIsEditing={setIsAdding}
-              isSavingEdits={isSavingEdits}
-              setIsSavingEdits={setIsSavingEdits}
-              saveEditedProperty={(_oldKey, newPair) =>
-                saveEditedCustomProperties(
-                  mergeUpdatedProperty({ customProperties, op: 'create', newPair }),
-                )
-              }
-            />
-          )}
-        </Tbody>
-      </Table>
-      {needExpandControl && (
-        <Button
-          variant="link"
-          className={spacing.mtSm}
-          data-testid="expand-control-button"
-          onClick={() => setIsShowingMoreProperties(!isShowingMoreProperties)}
+      <ExpandableSectionToggle
```

Comment:
Please reuse https://github.com/kubeflow/model-registry/blob/main/clients/ui/frontend/src/app/pages/modelRegistry/screens/components/ModelPropertiesExpandableSection.tsx here

Reply:
@Taj010 Hi, did you check this comment?